verify-ui/src/components/Home/VerificationSection/QrScanner.tsx

```diff
@@ -16,6 +19,8 @@ function QrScanner({ onClose, scannerActive }: {
   const dispatch = useAppDispatch();
   const [isCameraBlocked, setIsCameraBlocked] = useState(false);
   const [isScanning, setIsScanning] = useState(false);
+    const currentLang = i18next.language;
+    const [details, setDetails] = useState<{ key: string; value: string }[]>([]);
```

Comment:
What is the purpose of details, it was set but never used anywhere in the component